src/AppBundle/Converter/TwigTemplateConverter.php

```diff
@@ -220,9 +220,12 @@ private function convertLayout($mjmlTheme, $newTheme, $isWrapped)
         }
 
         //Each converted template has its own style rules, so we need to extract them as well
-        $templateStyles = $this->extractHtml($convertedTemplate, 'head style');
-        $templateStyles = trim($templateStyles)."\n";
-
+        $htmlHead = $this->extractHtml($convertedTemplate, 'head');
+        if (preg_match('#(<style.*</style>)#s', $htmlHead, $matches)) {
+            $templateStyles = trim($matches[1])."\n";
+        } else {
+            $templateStyles = '';
+        }
```

Comment:
Extracting the <style> tags is not sufficient. The HTML generated by mjml contains conditional styles (inside noscript and conditional comments) and a <link> tag.
We must include everything between the first and the last <style> tag.